### PR TITLE
Make DataPrepper work with new env

### PR DESCRIPTION
**Pull request by peterzhuamazon**
*Issue #, if available:*
V311401608

*Description of changes:*
This PR is to make DataPrepper work with new env

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/538405093

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/prod-sync-data-prepper.yml [hunk 1]
```diff
@@ -21,13 +21,14 @@ jobs:
     - name: S3 sync artifacts
       run: |
         #!/bin/bash
```
Comment (sreekarjami) [inline .github/workflows/prod-sync-data-prepper.yml:23]: We can remove this, as git actions assigns the bash as default.

File: .github/workflows/staging-build-data-prepper.yml [hunk 2]
```diff
@@ -16,23 +16,24 @@ jobs:
...
+        aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
         aws-region: us-east-1
 
+    - name: required utilites
+      run: release-tools/scripts/required_packages.sh
+
     - name: S3 sync artifacts
       run: |
         #!/bin/bash
```
Comment (sreekarjami) [inline .github/workflows/staging-build-data-prepper.yml:28]: We can remove this line, as git actions assign bash by default.

File: .github/workflows/prod-sync-data-prepper.yml (hunk 1 above)
Comment (peterzhuamazon) [reply .github/workflows/prod-sync-data-prepper.yml]: Removed.

File: .github/workflows/staging-build-data-prepper.yml (hunk 2 above)
Comment (peterzhuamazon) [reply .github/workflows/staging-build-data-prepper.yml]: Removed

**Review by sreekarjami — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

**Pull request merged**